code/modules/events/spacevine.dm

```diff
@@ -145,17 +147,50 @@
 	severity = 3
 	quality = NEGATIVE
 
-/datum/spacevine_mutation/aggressive_spread/on_spread(obj/structure/spacevine/holder, turf/target)
-	switch(severity)
-		if(EXPLODE_DEVASTATE)
-			SSexplosions.highturf += target
-		if(EXPLODE_HEAVY)
-			SSexplosions.medturf += target
-		if(EXPLODE_LIGHT)
-			SSexplosions.lowturf += target
+/datum/spacevine_mutation/aggressive_spread/on_spread(obj/structure/spacevine/holder, turf/target, mob/living)
+	for(var/mob/living/M in range(0,get_turf(target))) //Targets and checks all mobs on target turf individually.
```

Comment:
Why range(0) ? Just iterate mobs in the the turf directly.

Reply:
That was silly. Fixed.